router/userRouter.js

```diff
@@ -18,7 +18,7 @@ router.get("/:userId", userController.getUser);
 // register a user
 router.post("/", validate(registerRules), userController.addUser);
 // login a user --> req.body will contain email/password
-router.post("/login", userController.loginUser); // --> after this redirect to /users/:userId/events/today ?
+router.post("/login", validate(loginRules), userController.loginUser); // --> after this redirect to /users/:userId/events/today ?
```

Comment:
That is a good question and I guess yes...? Once logged in the redirect goes to the equivalent to landing page, is landing page is events today, it should be the same but showing the favorites (hearts and number of hearts) and then surf from therein modus user-logged-in.

Reply:
we will get there next week :)